packages/store-ui/src/atoms/TextArea/stories/TextArea.mdx

```diff
@@ -0,0 +1,12 @@
+import { Story, Canvas, ArgsTable } from '@storybook/addon-docs'
+import TextArea from '../TextArea'
+
+# Preview
+
+<Canvas>
+  <TextArea id="atoms-textarea--textarea" />
```

Comment:
is this id required?

```suggestion
  <TextArea />
```

Reply:
Ain't sure, I will try to remove it.

Reply:
I removed it, seems not to change anything.